api/dataloader/mediaURLLoader.go

```diff
@@ -56,7 +56,9 @@ func NewHighresMediaURLLoader(db *gorm.DB) *MediaURLLoader {
 		maxBatch: 100,
 		wait:     5 * time.Millisecond,
 		fetch: makeMediaURLLoader(db, func(query *gorm.DB) *gorm.DB {
-			return query.Where("purpose = ? OR (purpose = ? AND content_type IN ?)", models.PhotoHighRes, models.MediaOriginal, media_type.WebMimetypes)
+			return query.
```

Comment:
The format of this line is weird. Do you use `gofmt`?

In the other hand, I don't see good benefit to limit the size of a line. Morden editor can handle a long line well.

Reply:
Well, maybe I'm too old) But to me, long unsplit lines look not easily readable. Even with IDE's word wrapping.

I started my programming journey with Pascal/Delphi, then switched to Java, then to Python. Now getting familiar with Go. I used to see a visual code structure and even when dealing with long OOP calls or Java stream processing, I usually put each item of the chain to its own line with indention, representing the dependency level. It helps me to easily see the code structure.

BTW, I executed `gofmt ./api/dataloader/mediaURLLoader.go` and it returned me exactly the same formatting:
```
func NewHighresMediaURLLoader(db *gorm.DB) *MediaURLLoader {
        return &MediaURLLoader{
                maxBatch: 100,
                wait:     5 * time.Millisecond,
                fetch: makeMediaURLLoader(db, func(query *gorm.DB) *gorm.DB {
                        return query.
                                Where("purpose = ? OR (purpose = ? AND content_type IN ?)",
                                        models.PhotoHighRes, models.MediaOriginal, media_type.WebMimetypes)
                }),
        }
}
```

Reply:
Oh, ok. I misreaded it as spaces. LGTM.